regression/contracts/invar_loop_constant/test.desc

```diff
@@ -1,9 +1,10 @@
 KNOWNBUG
 main.c
---check-code-contracts
+--enforce-all-contracts
 ^EXIT=0$
 ^SIGNAL=0$
 ^VERIFICATION SUCCESSFUL$
 --
 --
-Loop invariant checking throws away more information than needed.
+The current implementation of `check_apply_invariants` function
```

Comment:
What is the exact behavior you expect for this test case?

Reply:
I clarified the description for this test and added a couple more related tests to demonstrate the expected behavior.